src/DoctrineBundle.php

```diff
@@ -168,4 +170,14 @@ public function shutdown()
     public function registerCommands(Application $application)
     {
     }
+
+    public function getPath(): string
+    {
+        /** @psalm-suppress RedundantPropertyInitializationCheck */
+        if (! isset($this->path)) {
```

Comment:
I would even not care about assigning the `path` property. This is not needed (the base class only uses it as a cache for its `getPath` implementation that relies on Reflection)